trello/1.1/modules/ROOT/pages/trello-connector-reference.adoc

```diff
@@ -23713,6 +23787,83 @@ Update Webhooks Id Model By Id Webhook. This operation makes an HTTP PUT request
 * TRELLO:UNSUPPORTED_MEDIA_TYPE
 
 
+== Sources
+
+[[OnNewCardTrigger]]
+== On New Card Trigger
+`<trello:on-new-card-trigger>`
+
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use | | x
+| Organization ID a| String |  ID of the organization |  |
+| Since timestamp a| String |  Timestamp of since when to query new or updated cards |  |
+| Config Ref a| ConfigurationProvider |  Name of the configuration used to execute this component |  |
+| Primary Node Only a| Boolean |  Whether this source should only be executed on the primary node when running in Cluster |  |
```

Comment:
There is a description for this in the Reference spreadsheet: https://docs.google.com/spreadsheets/d/1FzjhPtRWNHmaNK_8JbXbkbblZiul6zELQ2I4Hys0_UI/edit#gid=0